docs/manual/annotating-libraries.tex

```diff
@@ -537,7 +537,10 @@
 bytecode annotations.
 
 A file being compiled takes precedence over a stub file.  If file \<A.java>
-is being compiled, then any stub for class \<A> is ignored.
+is being compiled, then any stub for class \<A> is ignored. There is a
+command-line flag that can be provided to the Checker Framework to use
```

Comment:
This option doesn't just "use" the stub file, it picks the stronger annotation.  Would you please clarify this?